docs/design/index.md

```diff
@@ -22,79 +22,74 @@ title: "Introduction to Apache Druid"
-4. **Realtime or batch ingestion.** Druid can ingest data either real-time (ingested data is immediately available for
-querying) or in batches.
-5. **Self-healing, self-balancing, easy to operate.** As an operator, to scale the cluster out or in, simply add or
-remove servers and the cluster will rebalance itself automatically, in the background, without any downtime. If any
-Druid servers fail, the system will automatically route around the damage until those servers can be replaced. Druid
-is designed to run 24/7 with no need for planned downtimes for any reason, including configuration changes and software
+1. **Columnar storage format.** Druid uses column-oriented storage. This means it only loads the exact columns
+needed for a particular query.  This greatly improves speed for queries that retrieve only a few columns. Additionally, to support fast scans and aggregations, Druid optimizes column storage for each column according to its data type.
+2. **Scalable distributed system.** Typical Druid deployments span clusters ranging from tens to hundreds of servers. Druid can ingest data at the rate of millions of records per second while retaining trillions of records and maintaining query latencies ranging from the sub-second to a few seconds.
+3. **Massively parallel processing.** Druid can process each query in parallel across the entire cluster.
+4. **Realtime or batch ingestion.** Druid can ingest data either real-time or in batches. Ingested data is immediately available for
+querying.
+5. **Self-healing, self-balancing, easy to operate.** As an operator, you add servers to scale out or
+remove servers to scale down. The Druid cluster re-balances itself automatically in the background without any downtime. If a
+Druid server fails, the system automatically routes data around the damage until the server can be replaced. Druid
+is designed to run continuously without planned downtime for any reason. This is true for configuration changes and software
 updates.
-6. **Cloud-native, fault-tolerant architecture that won't lose data.** Once Druid has ingested your data, a copy is
-stored safely in [deep storage](architecture.md#deep-storage) (typically cloud storage, HDFS, or a shared filesystem).
-Your data can be recovered from deep storage even if every single Druid server fails. For more limited failures affecting
-just a few Druid servers, replication ensures that queries are still possible while the system recovers.
+6. **Cloud-native, fault-tolerant architecture that won't lose data.** After ingestion, Druid safely stores a copy of your data in [deep storage](architecture.md#deep-storage). Deep storage is typically cloud storage, HDFS, or a shared filesystem. You can recover your data from deep storage even in the unlikely case that all Druid servers fail. For a limited failure that affects only a few Druid servers, Druid uses replication to ensure that queries are still possible during system recovers.
```

Comment:
I think "recovers" -> "recoveries", although the original seems to refer a little more clearly to "For a limited failure that affects..":

```suggestion
6. **Cloud-native, fault-tolerant architecture that won't lose data.** After ingestion, Druid safely stores a copy of your data in [deep storage](architecture.md#deep-storage). Deep storage is typically cloud storage, HDFS, or a shared filesystem. You can recover your data from deep storage even in the unlikely case that all Druid servers fail. For a limited failure that affects only a few Druid servers, replication ensures that queries are still possible during system recoveries.
```